AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -638,7 +648,7 @@ - (void)rangeControllerBeginUpdates:(ASRangeController *)rangeController {
 - (void)rangeController:(ASRangeController *)rangeController endUpdatesAnimated:(BOOL)animated completion:(void (^)(BOOL))completion {
   ASDisplayNodeAssertMainThread();
 
-  if (!self.asyncDataSource) {
+  if (!self.asyncDataSource || _pendingReloadData) {
     if (completion) {
       completion(NO);
```

Comment:
Should we call `completion(YES)` if we suppressed the update due to a pending reload?